types/katex.d.ts

```diff
@@ -20,6 +20,43 @@ export type TrustContext =
     | { command: "\\htmlStyle", style: string }
     | { command: "\\htmlData", attributes: Record<string, string> }
 
+
+type KatexCatcodes = Record<string, number>;
+
+interface KatexLexerInterFace {
```

Comment:
I also don't think you need `Interface` in the name of this `Interface`. Matching the internal `Lexer` seems fine.